scss/_accordion.scss

```diff
@@ -115,3 +115,39 @@
     letter-spacing: $accordion-button-letter-spacing-lg;
   }
 }
+
+// Boosted mod
+.accordion-dark {
```

Comment:
like button are button-inverse maybe use accordion-inverse instead of dark?

Reply:
I don't really know about that, I think that `.btn-inverse` is there because there is already a `.btn-dark` variant.

Reply:
Indeed `.btn-inverse` has this name because `.btn-dark` is already taken.
We thought calling dark variants of Boosted components `.component-dark` because Bootstrap uses it for the [Carousel - Dark variant](https://getbootstrap.com/docs/5.1/components/carousel/#dark-variant) with their `.carousel-dark`.